zerver/webhooks/uptimerobot/view.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any, Dict
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import api_key_only_webhook_view
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_success
+from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.models import UserProfile
+
+templates = {
+    'Up': (u'The monitor {monitorFriendlyName} ({monitorURL}) is back UP ({alertDetails}).' +
+           u' It was down for {alertFriendlyDuration}.'),
```

Comment:
The `u`s here are unnecessary and should be removed; they're a Python 2/3 transition relic.

Reply:
@timabbot now I understand why I didn't see it in recent codes. Accordingly removed.

Reply:
Forgot to add before commiting. Fixed now.

Reply:
@timabbot test failed randomly. All tests passed after pushing again.